GOVERNANCE.md

```diff
@@ -1,114 +1,287 @@
+as security exploits, CoC enforcement, and legal questions.
+
+Steering Committee members are expected to advocate for all of Operator
+Framework, not just certain projects or corporate sponsors, comply with
+and support the Code of Conduct, and be professional and compassionate
+in all of their dealings with project participants.
+
+### Steering Committee Elections
+
+Once per year, the Steering Committee will select three Election
+Officers to run the annual election and set the dates for the election.
+Election Officers should be project Members in good standing, not
+running for election themselves, and represent more than one project
+employer. These officers will update the list of eligible Members according to
+project records, send out announcements, and conduct the election.
+
+The election itself will last for at least one week, and is conducted as
+a preference election online, using the Condorcet method. The Election
+Officers will announce the selected candidates at the next regular
+community meeting.
+
+#### Steering Committee Election Procedure 
+
+1. Nominations for Operator Framework Representatives (Subproject Rep
+   and Community Rep) will be collected for at least 14 days. Interested
+   individuals should self-nominate, and they may be considered for
+   Subproject Representative and Community Representative.  
+2. For each Subproject, one Subproject Representative is chosen by the
+   Subproject's Approvers. Each Subproject vote will be conducted by the
+   OF Election Officials and calculated by the Condorcet Method.
+3. Following the announcement of the Subproject results, the
+   Community Representatives are chosen by all Members of the Operator
+   Framework. The vote will be conducted by the OF Election Officials
+   and calculated by the Condorcet Method.
+
+If individuals are nominated to be Subproject Representative, the Subproject elections will be calculated in a random order. No individual can
+hold more than one seat on the Steering Committee.
+
+### Steering Committee Bootstrapping
+
+Upon ratification of this document, the 7 elected Chairs of the Operator
+Framework will become the Steering Committee for a period of one year,
+afterwards Steering Committee elections will proceed as outlined in this
+document.
+
+## Governance Amendments
+
+The Operator Framework Community may adopt or amend its Governing
+Documents according to the following process:
+
+1. Propose a change by opening an issue in the community, and if there
+   seems to be general agreement, open a pull request to this document.
+2. The change should be announced via the mailing list along with an invitation for discussion on the Pull Request.
+3. A minimum of 7 days must pass between the announcement and the vote. A longer period can be requested with approval from the steering committee. All maintainers, approvers, subproject owners, and the Steering committee are eligible to vote.
```

Comment:
4. The vote must remain open for a minimum of 7 days, the Steering Committee may extend the period.